src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -25,11 +61,84 @@ def title
       end
 
       def contents
-        VBox(SizeWidget.new(@disk, @ptemplate, @slots))
+        HVSquash(SizeWidget.new(@disk, @ptemplate, @slots))
+      end
+
+      # Like CWM::RadioButtons but each RB has a subordinate indented widget.
+      # This is kind of like Pager, but all Pages being visible at once,
+      # and enabled/disabled.
+      # Besides `items` there are also `widgets`
+      class ControllerRadioButtons < CWM::CustomWidget
+        def initialize
+          self.handle_all_events = true
+        end
+
+        def contents
+          Frame(
+            label,
+            MarginBox(
+              hspacing, vspacing,
+              RadioButtonGroup(Id(widget_id), buttons_with_widgets)
+            )
+          )
+        end
+
+        # @return [Array<Array(String,String)>]
+        abstract_method :items
+
+        # FIXME: allow {WidgetTerm}
+        # @return [Array<AbstractWidget>]
+        abstract_method :widgets
+
+        def hspacing
+          1.45
+        end
+
+        def vspacing
+          0.45
+        end
```

Comment:
my interest is why it is not constant this quite strange numbers ( anything different then 0 a 1 should be explained :)